ngraph_bridge/ngraph_enter_in_catalog.cc

```diff
@@ -0,0 +1,83 @@
+/*******************************************************************************
+ * Copyright 2019 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
```

Comment:
we discussed about calling this file enter_prefetch_in_catalog

Reply:
I don't remember discussing the name of the file. I was under the impression that ultimately both the enter_in_catalog files will become one.

Reply:
Main objective for that was to touch only ngraph_catalog files and not the enter in catalog files, and enter prefetch in catalog is valid for both builds